rasa/core/channels/channel.py

```diff
@@ -38,8 +38,6 @@ def __init__(
     ) -> None:
         if text:
             self.text = text.strip()
-        else:
-            self.text = text
```

Comment:
`self.text` needs to be set to some value, otherwise it will be undefined in other functions

Reply:
Is this another one of those initialization things like "don't initialize to `[]` or `{}`"? I thought it would be set to `None` [here](https://github.com/RasaHQ/rasa/pull/3478/files#diff-3711937129d5f7a303644cd9a60cf55dR32) instead of in L42

Reply:
ahh i see my bad, we're talking about `self.text`, not `text`. Makes sense, will revert